main.cpp

```diff
@@ -359,7 +464,23 @@ int main()
     FloatType floatType(3.1f);
     IntType intType(2);
     DoubleType doubleType(3.1);
-    
+
+    // Float Type: Apply() passing a lambda
+
+    floatType.apply([&ft = floatType](float& f) -> FloatType&
+    {
+        f *= 8.f;
+        return ft;
+    });
```

Comment:
I've never seen anyone capture a reference like this and rename the reference.
Did you do this just to do it, or to improve readability?